server/config/config.go

```diff
@@ -201,7 +201,7 @@ func Convert(cfg *Config) *config.Config {
 		"frontend": cfg.mustGetService(0),
 		"history":  cfg.mustGetService(1),
 		"matching": cfg.mustGetService(2),
-		"worker":   cfg.mustGetService(3),
+		"worker":   cfg.mustGetService(6),
```

Comment:
Revert this?

Reply:
sure (took the value it from temporal server)

Reply:
reverted

though i wouldn't mind having the same as the default params in Temporal server repo (to address a niche corner case when Temporal repo server starts and this one would complain because of a taken port)